packages/notebook/src/model.ts

```diff
@@ -243,6 +245,42 @@ export class NotebookModel extends DocumentModel implements INotebookModel {
       this._nbformatMinor = newValue = value.nbformat_minor;
       this.triggerStateChange({ name: 'nbformatMinor', oldValue, newValue });
     }
+
+    // Alert the user if the format changes.
+    if (origNbformat !== undefined && this._nbformat !== origNbformat) {
+      let src;
+      if (this._nbformat > origNbformat) {
+        src = ' an older notebook format ';
+      } else {
+        src = ' a newer notebook format ';
+      }
+
+      let msg =
```

Comment:
It would be much cleaner to use a backtick string here to interpolate variables.

Reply:
Modernized now :wink: